superset-frontend/src/dashboard/components/nativeFilters/FilterBar/FilterBarCrossFilters/CrossFilter.tsx

```diff
@@ -0,0 +1,229 @@
+/**
```

Comment:
That's a really minor nitpick but... since we're already in `FilterBar` directory, maybe we should name the subdirectory `CrossFilters` instead of `FilterBarCrossFilters`?